pizzakit-plugin/includes/pizzakit.php

```diff
@@ -242,11 +242,10 @@ function($curl, $header) use (&$headers) {
 	private static function insert_into_tables($_data)
 	{
 
-		$json = file_get_contents(plugin_dir_path(__FILE__) . 'items_for_sale.json');
-		$decoded = json_decode($json, true);
-		$items = $decoded['menu'];
-
 		global $wpdb;
+		$sql = "SELECT * FROM wp_items";
+		$items = $wpdb->get_results($sql,$output=ARRAY_N);
+		trigger_error(print_r($items,$return=true));
```

Comment:
Should we keep debug messages in production code?

Reply:
While i would advocate that we should add some logs to help debugging, this is obviously not a good way to do this